emr_launcher.py

```diff
@@ -130,14 +130,14 @@ def handler(event: dict = {}, context: object = None) -> dict:
 
     # If when this lambda is triggered via API
     # Else when this lambda is triggered via SNS
-    if CORRELATION_ID in event and S3_PREFIX in event:
-        correlation_id = event[CORRELATION_ID]
-        s3_prefix = event[S3_PREFIX]
+    if PAYLOAD_S3_PREFIX in event and PAYLOAD_S3_PREFIX in event:
```

Comment:
I think you meant to check both S3_PREFIX and CORRELATION_ID here but checking for S£_PREFIX twice by mistake

Reply:
just identified it and pushed it :)